lib/wiznotesDisplay.ml

```
@@ -634,6 +634,8 @@ let connected_wizards conf base =
 
 let do_change_wizard_visibility conf base x set_vis =
   let wddir = dir conf base in
+  if Sys.file_exists wddir then () else
```

```
if not @@ Sys.file_exists wddir then Unix.mkdir wddir 0o755
```

(Do not silently fail, and avoid useless `()` action in a `if`)